### PR TITLE
New video! (Workflow 2023)

### PR DESCRIPTION
**Pull request by shiffman**
Adding to start of workflow track for now, eventually will go into a new track see #968

### CODE REVIEW

**Comment by shiffman** [comment]
Thanks for catching the mistakes @dipamsen!!

File: content/videos/workflow/2023-update/index.json [hunk 1]
```diff
@@ -0,0 +1,136 @@
+{
+  "title": "Workflow: Terminal, Shell, Node.js, VSCode",
+  "description": "This video covers tools and configurations to set up a  JavaScript development environmnt with Node.js, Visual Studio Code, and shell/terminal access. This is step 1 along the way to creating your own Discord Bot!",
```
Comment (supercrafter100) [inline content/videos/workflow/2023-update/index.json:3]: Got two spaces between "a" and "JavaScript"

Comment (supercrafter100) [reply content/videos/workflow/2023-update/index.json:3]: Additionally, environmnt is misspelled

**Review by supercrafter100 — CHANGES_REQUESTED** [review]

**Comment by shiffman** [comment]
Merging and video going live momentarily!

**Pull request merged**